### PR TITLE
MGDDOC-55: Test case for adding BYOC info for RHOAM

### PR DESCRIPTION
**Pull request by joaedwar**
# Description

Test published AWS SCP info for BYOC in a KCS article.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [x] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: test-cases/tests/documentation/i06-aws-info-byoc.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+products:
+  - name: rhoam
+    environments:
+      - osd-fresh-install
+    targets:
+      - 0.1
```
Comment (psturc) [inline test-cases/tests/documentation/i06-aws-info-byoc.md:7]: If this test case should be performed as a part of RHOAM GA testing, we should target version 1.0 instead:
```suggestion
      - 1.0
```

**Review by psturc — COMMENTED** [review]
Hey @joaedwar, thanks for adding the new test case!

I've added one suggestion.

Also `test-cases-lint` job is failing, which can be fixed by running
```
npm install && npm run prettier && npm run lint
```
from the `test-cases` folder and by committing and pushing a new changes

**Comment by joaedwar** [comment]
Hi @pstruc I ran that command along with the `npm rename` and `npm prettier`. Is it ok to go now?

**Comment by psturc** [comment]
@joaedwar the prow job is still failing:
Screenshot 2020-11-19 at 11 29 11

can you try it again and push the new changes? If it doesn't work for you, ping me and we can sync and sort this out

**Comment by joaedwar** [comment]
@psturc can we sync up to sort this out?

**Comment by joaedwar** [comment]
@psturc I am still getting the message. I am not sure what I am doing wrong

**Comment by psturc** [comment]
@joaedwar tests have actually passed for this PR now ;) it's good to merge

/lgtm
/approve

**Pull request merged**